python/tracker.py

```
@@ -16,7 +16,7 @@
 cuVSLAM Tracker - A wrapper combining Odometry and SLAM functionality.
 """
 
-from typing import Any, Optional, List, Callable, Tuple
+from typing import Any, Optional, List, Sequence, Callable, Tuple
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Sequence`, `Callable`

Import from `collections.abc`

(UP035)